src/gtfs_structures.rs

```diff
@@ -49,8 +49,8 @@ pub struct CalendarDate {
 pub struct Stop {
     #[serde(rename = "stop_id")] pub id: String,
     pub stop_name: String,
-    location_type: Option<u8>,
-    parent_station: Option<String>,
+    pub location_type: Option<u8>,
```

Comment:
what is the location_type ?

Reply:
https://developers.google.com/transit/gtfs/reference/#stopstxt

Reply:
It can be a `u8` as the default value is 0

Reply:
my point was that it was easier to understand if it was an enum.